packages/manager/apps/account-creation/src/data/hooks/settings/useSettings.ts

```diff
@@ -71,3 +71,30 @@ export const useLanguageSettings = (country?: string, currency?: string) =>
       );
     },
   });
+
+export const useSubsidiarySettings = (
+  country?: string,
+  currency?: string,
+  language?: string,
+) => {
+  return useQuery({
+    queryKey: ['settings'],
```

Comment:
Can we move the query key inside data/api/setting folder as getSettingsQueryKey 
export const getSettingsQueryKey = (region) => ['/account/settings']; as of now.
And use this once API is ready 
export const getSettingsQueryKey = (region) => ['/account/settings?ovhIS=${region}'];
```suggestion
    queryKey: getSettingsQueryKey,
```